app/auth/routes.py

```
@@ -15,6 +15,31 @@
 
+@router.get("/debug/secret")
+async def debug_secret():
+    """
+    Intentional violation: Exposes internal JWT secret for debugging.
+    """
+    
+    return {"jwt_secret": firebase_auth.jwt_secret}
+
+
+@router.get("/unsafe-login")
+async def unsafe_login(email: str = None, password: str = None):
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)